src/en/ss14-by-example/adding-a-simple-bikehorn.md

```diff
@@ -145,7 +145,7 @@ Those helpers can be thought of as handling **client click -> server** and **ser
 In the Space Station 14 codebase, Components & EntitySystems alike (along with other classes) go inside folders directly under the `Content.Server`, `Content.Shared`, or `Content.Client` projects. There are folders for `Atmos`, `Botany`, `Research`, `Storage`, and a lot more. If a suitable folder doesn't exist, create one! Never put files directly into the top directory of the project.
 ```
 
-Under the `Content.Server` project, there's a folder called `Sound`. That seems like a good place to put our new component (and in fact, this is where the real EmitSoundOnUse is located). Let's call our version `PlaySoundOnUseComponent`. Note: if you just copy paste this code in, it may not work, as you'll need to import various classes. Your IDE can do this for you.
+Under the `Content.Server` project, there's a folder called `Sound`. This folder contains an apty named `Components` folder. That seems like a good place to put our new component (and in fact, this is where the real `EmitSoundOnTriggerComponent` is located). Let's call our version `PlaySoundOnUseComponent`. Note: if you just copy paste this code in, it may not work, as you'll need to import various classes. Your IDE can do this for you.
```

Comment:
What is an apty? I don't think that's what you meant to write?

Reply:
aptly

Reply:
whoops, my bad